google/auth/credentials.py

```diff
@@ -469,6 +483,13 @@ def signer(self):
 
 
 class TokenState(Enum):
+    """
+    Tracks the state of a token.
+    FRESH: The token is not expired and can be used normally.
```

Comment:
The token is valid.  It is not expired or close to expired, or the token has no expiry.  To make it mutually exclusive to STALE.